src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderMessages/RequiredStakeMessage.tsx

```diff
@@ -3,13 +3,8 @@ import { useFormContext } from 'react-hook-form';
 import { defineMessages, FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 
-import { useColonyContext } from '~hooks';
 import Numeral from '~shared/Numeral';
-import {
-  getStakeFromSlider,
-  getStakePercentage,
-  useStakingWidgetContext,
-} from '../..';
+import { getStakeFromSlider, getStakePercentage } from '../..';
```

Comment:
Uff. Can we please have these named, as when going up the tree this gets really confusing